Sources/ActionSheet/ActionSheetSortSectionHeader.swift

```diff
@@ -30,66 +30,66 @@ class ActionSheetSortSectionHeader: ActionSheetSectionHeader {
     private let userDefaults = UserDefaults.standard
 
     private let descendingStackView: UIStackView = {
-        let descendingStackView = UIStackView()
-        descendingStackView.spacing = 0
-        descendingStackView.alignment = .center
-        descendingStackView.translatesAutoresizingMaskIntoConstraints = false
-        return descendingStackView
+        let stackView = UIStackView()
```

Comment:
only this `UIStackView` is actually `descendingStackView `. Others are copy-pasted. I think that basic `stackView` is better for copy-pasting ;-)
The same logic for other similar renames in closures for `let` value calculations